script/tool/lib/src/fetch_deps_command.dart

```diff
@@ -63,12 +169,81 @@ class FetchDepsCommand extends PackageLoopingCommand {
 
       final String packageName = package.directory.basename;
 
-      final int exitCode = await gradleProject.runCommand('$packageName:dependencies');
+      final int exitCode =
+          await gradleProject.runCommand('$packageName:dependencies');
       if (exitCode != 0) {
         return PackageResult.fail();
       }
     }
 
     return PackageResult.success();
   }
+
+  Future<PackageResult> _fetchDarwinDeps(
+      RepositoryPackage package, final String platform) async {
+    if (!pluginSupportsPlatform(platform, package,
+        requiredMode: PlatformSupport.inline)) {
+      final String displayPlatform = platform.replaceFirst('os', 'OS');
```

Comment:
consider adding a comment for why this replacement is required.

Reply:
Done.